tiledb/sm/query/reader.cc

```diff
@@ -3212,6 +3212,11 @@ Status Reader::copy_attribute_values(
     const std::vector<ResultCellSlab>& result_cell_slabs) {
   STATS_START_TIMER(stats::Stats::TimerType::READ_COPY_ATTR_VALUES);
 
+  if (result_cell_slabs.empty() || result_tiles.empty()) {
```

Comment:
Could we also add this check to `Reader::copy_coordinates`?

This would also obsolete the `if (result_cell_slabs.empty()) {` checks in `Reader::copy_fixed_cells` and `Reader::copy_var_cells`. It's probably safer to leave those.

Reply:
Added it to `Reader::copy_coordinates` but left the exits in `Reader::copy_fixed_cells` and `Reader::copy_var_cells`